website/docs/reference/commands/source.md

```diff
@@ -12,6 +12,64 @@ The `dbt source` command provides subcommands that are useful when working with
 
 If your dbt project is [configured with sources](/docs/build/sources), then the `dbt source freshness` command will query all of your defined source tables, determining the "freshness" of these tables. If the tables are stale (based on the `freshness` config specified for your sources) then dbt will report a warning or error accordingly. If a source <Term id="table" /> is in a stale state, then dbt will exit with a nonzero exit code.
 
+You can also use [source freshness](/docs/deploy/source-freshness) commands help make sure the data you get is new and not old or outdated. 
+
+### Configure source freshness
```

Comment:
very cool! this seems a little duplicative/repetitive of [this page though](https://docs-getdbt-com-git-nfiann-sourcefreshness-dbt-labs.vercel.app/docs/build/sources#declaring-source-freshness) -- should we not just link to docs/build/sources page here?